.circleci/config.yml

```diff
@@ -162,7 +162,18 @@ jobs:
             cf api https://api.fr.cloud.gov
             cf auth "$CF_USERNAME" "$CF_PASSWORD"
             cf target -o $CF_ORG -s "$CF_SPACE"
-            ./deploy-cloudgov-docker.sh rolling test $CGHOSTNAME_BACKEND $CGHOSTNAME_FRONTEND $DOCKER_IMAGE_BACKEND $DOCKER_IMAGE_FRONTEND $CIRCLE_BRANCH
+
+            # Deploy ClamAV REST server
+            # First delete the existing implementation of clamav-rest so we don't exhaust memory quota
+            cf delete -r clamav-rest -f
+            # NOTE: docker-backend is unused by clamav-rest but is needed to prevent an error when calling cf push
+            cf push clamav-rest -f tdrs-backend/manifest.yml --var docker-backend=null --var cf-space=$CF_SPACE
+
+            # Deploy frontend and backend applications
+            ./deploy-cloudgov-docker.sh rolling test $CGHOSTNAME_BACKEND $CGHOSTNAME_FRONTEND $DOCKER_IMAGE_BACKEND $DOCKER_IMAGE_FRONTEND $CIRCLE_BRANCH $CF_SPACE
+
+            # Enable direct communication over internal routes between backend and clamav-rest containers
+            cf add-network-policy $CGHOSTNAME_BACKEND clamav-rest -s $CF_SPACE -o $CF_ORG --protocol tcp --port 9000
```

Comment:
This was needed to gain access to the non-standard port `9000` from within the `tdp-backend` application.